src/ontology/cl-edit.owl

```diff
@@ -9536,14 +9536,17 @@ SubClassOf(obo:CL_0000705 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000097
 
 # Class: obo:CL_0000706 (choroid plexus epithelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:9550134") obo:IAO_0000115 obo:CL_0000706 "Specialized ependymal cell that produces the cerebrospinal fluid from the blood and secretes it into the lumen of the brain and spinal chord.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:15561411") Annotation(oboInOwl:hasDbXref "PMID:9550134") obo:IAO_0000115 obo:CL_0000706 "A specialized ependymal cell that is part of the choroid plexus epithelium, responsible for producing cerebrospinal fluid (CSF) from the blood and secreting it into the brain and spinal cord. The cell is characterized by a brush border on its apical surface, which enhances the secretion of CSF.")
```

Comment:
In my opinion, "producing cerebrospinal fluid (CSF) from the blood" is a bit vague. 
If you agree, consider "producing cerebrospinal fluid (CSF) by selectively filtering and modifying blood plasma components".

Reply:
I agree